demo/app/components/confirmation/confirmation.component.html

```diff
@@ -7,6 +7,7 @@
     (cancelled)="cancel($event)"
     confirmationButtonText="Custom Confirmation Button Text"
     [explanationText]="explanation"
+    modalPosition="above"
```

Comment:
True.. we could add a dropdown or radio buttons to change the side. I think the tooltip demo does this

Reply:
Side note.. if we add the ability to change the side dynamically, we will probably want to use the same property to adjust the layout of the demo. That way when `after` is selected the button moves to the far left of the screen (using flexbox) so that it's a better representation of when `after` would be used

Reply:
Great suggestion!